<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestiondereservation</class>
 <widget class="QDialog" name="gestiondereservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1584</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1511</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1511</width>
      <height>538</height>
     </rect>
    </property>
    <attribute name="label">
     <string>gestion reservation</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1061</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter reservation</string>
      </attribute>
      <widget class="QPushButton" name="pushButtonajouterreservation">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>470</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelidreservation">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant de reservation : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditidreservation">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>60</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelcinclient">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN du client : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditcinclient">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labeldatereservation">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de reservation : </string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>150</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelsallereserver">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Salle à reserver : </string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxsalle">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>200</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelphotographe">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>250</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Photographe à reserver : </string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxphotographe">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>250</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelnote">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>370</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Note du couple : </string>
       </property>
      </widget>
      <widget class="QLabel" name="labelnombreinvite">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>290</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre d'invitees</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelbudget">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>330</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>budget : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditbudget">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>330</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditnbinvite">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>290</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>360</y>
         <width>291</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>affichier reservation</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>70</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 170, 127);</string>
       </property>
       <property name="text">
        <string>detaile</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>180</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>240</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>archiver</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>300</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>851</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>120</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>trier </string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1511</width>
      <height>538</height>
     </rect>
    </property>
    <attribute name="label">
     <string>gestion client</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1511</width>
       <height>551</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Ajouter client </string>
      </attribute>
      <widget class="QPushButton" name="pushButtonajouter">
       <property name="geometry">
        <rect>
         <x>552</x>
         <y>470</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>122</x>
         <y>22</y>
         <width>301</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelcin">
          <property name="text">
           <string>CIN : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditcin"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>121</x>
         <y>83</y>
         <width>301</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelnom">
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditnom"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>121</x>
         <y>146</y>
         <width>301</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelprenom">
          <property name="text">
           <string>Prenom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditprenom"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>121</x>
         <y>270</y>
         <width>311</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labeladresse">
          <property name="text">
           <string>Adresse : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditadresse"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>121</x>
         <y>332</y>
         <width>311</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelemail">
          <property name="text">
           <string>E-mail :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditemail">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>390</y>
         <width>311</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="labelnumero">
          <property name="text">
           <string>Numero de telephone :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditnumero"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>440</y>
         <width>311</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>sexe :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineeditsexe"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>311</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labeldate">
          <property name="text">
           <string>Date de naissance : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditdate"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>affichier client</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>imprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonsupprimerclient">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>0</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>1021</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineeditcinsup">
       <property name="geometry">
        <rect>
         <x>1290</x>
         <y>20</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>1150</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>donner le cin de client : </string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_41">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>70</y>
         <width>201</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_36">
        <item>
         <widget class="QLabel" name="labelnom_6">
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditnom_6"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_51">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>100</y>
         <width>201</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_45">
        <item>
         <widget class="QLabel" name="labelprenom_7">
          <property name="text">
           <string>Prenom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditprenom_7"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_52">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>130</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_46">
        <item>
         <widget class="QLabel" name="labeldate_7">
          <property name="text">
           <string>Date de naissance : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditdate_7"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_53">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>160</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_47">
        <item>
         <widget class="QLabel" name="labeladresse_7">
          <property name="text">
           <string>Adresse : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditadresse_7"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_54">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>190</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_48">
        <item>
         <widget class="QLabel" name="labelemail_7">
          <property name="text">
           <string>E-mail :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditemail_7">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_55">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>220</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_49">
        <item>
         <widget class="QLabel" name="labelnumero_7">
          <property name="text">
           <string>Numero de telephone :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditnumero_7"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_56">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>260</y>
         <width>281</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_50">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>sexe :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineeditsexe_7"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>370</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonchercher">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>10</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>chercher</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>cin croissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom croissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prenom croissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cin decroissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom decroissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prenom decroissant</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1511</width>
      <height>538</height>
     </rect>
    </property>
    <attribute name="label">
     <string>faire statistique </string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
